common/app/common/configuration.scala

```diff
@@ -19,52 +19,65 @@ import scala.util.{Failure, Success, Try}
 
 class BadConfigurationException(msg: String) extends RuntimeException(msg)
 
-object InstallVars {
+object Environment {
 
-  val installVars = new File("/etc/gu/install_vars") match {
+  private[this] val properties = {
+    def env(name: String): Option[String] = sys.env.get(name)
+
+    val props = new File("/etc/gu/install_vars") match {
       case f if f.exists => IOUtils.toString(new FileInputStream(f), Charset.defaultCharset())
       case _ => ""
     }
 
-  val properties = Properties(installVars)
+    for {
+      p <- Properties(props)
+    } yield p match {
```

Comment:
why a for with a single statement rather than a simple `flatMap`?

Reply:
Purely personal preference. I tend to find that they are generally more readable and easier to adapt/modify.

Reply:
e.g. versus

    Properties(props).map{ case (key, value) => key -> env(key).getOrElse(value) }